indico/modules/events/sessions/templates/display/session_list.html

```diff
@@ -0,0 +1,47 @@
+{% extends 'layout/conference_page_base.html' %}
+
+{% block title %}
+    {% trans %}My Sessions{% endtrans %}
+{% endblock %}
+
+{% block content %}
+    <table id="sessions-list" class="i-table-widget">
+        <thead>
+            <tr>
+                <th>{% trans %}Name{% endtrans %}</th>
+                <th></th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for sess in sessions %}
+                <tr>
+                    <td>
+                        {% if session.user and sess.can_manage(session.user) %}
+                            <a href="{{ url_for('.session_list', event) }}">{{ sess.title }}</a>
+                        {% else %}
+                            {{ sess.title }}
+                        {% endif %}
+                    </td>
+                    <td>
+                        <div class="group right">
+                            <a href="#" class="icon-time" title="{% trans %}Go to this session timetable{% endtrans %}"></a>
+                        </div>
+                    </td>
+                </tr>
+            {% else %}
+                <tr>
+                    <td colspan="2">
+                        {% trans %}You are not linked to any sessions at the moment{% endtrans %}
```

Comment:
That's rather weird wording. Why not *"You do not have any sessions at the moment"*?